GETTING_STARTED.md

```diff
@@ -0,0 +1,127 @@
+initial meetings around the image and runtime specification in 2015. The
+following subsections define project categories which exist in the OCI today.
+
+### Specifications
+
+Clearly the image, runtime, and distribution specifications are the key
+reason for the existence of the OCI today. These standards are meant to
+provide a baseline to follow for implementors of runtimes, registries, and
+tools which interact with container images and registries.
+ - [Runtime spec](https://github.com/opencontainers/runtime-spec)
+ - [Image spec](https://github.com/opencontainers/image-spec)
+ - [Distribution spec](https://github.com/opencontainers/distribution-spec)
+
+### Spec Conformance Test
+
+Conformance tests should provide a clear, end-user runnable test suite for
+implementors to use to determine and demonstrate that an implementing project
+or product meets the explicit definitions of the specification.
+ - [OCI Conformance](https://github.com/opencontainers/oci-conformance)
+
+The most advanced conformance implementation to date is for the new distribution specification. Additional work on image and runtime conformance is ongoing.
+
+### Libraries 
+
+While hosting library code is not a common goal of the OCI, there are a few
+specific cases where small, non-end user focused, and tightly scoped libraries
+have been accepted into the OCI. The common denominator for these libraries are
+that they help implementors properly use features of the specifications:
+ - [go-digest](https://github.com/opencontainers/go-digest)
+ - [selinux](https://github.com/opencontainers/selinux)
+
+Utilities and end-user UX-oriented code is most likely better targeted at other
+more broad communities like the [CNCF](https://cncf.io). While there are not
+explicit rules, a discussion with the TOB is warranted for projects looking to
+contribute a library to OCI.
+
+### Reference Implementations
+
+While theoretically a specification can have one or more reference
+implementations, the OCI runtime spec and the program, `runc`, have gone
+hand in hand simply due to the particulars around the founding of the OCI.
+
+It is not inconceivable that more reference implementations would be 
+contributed and supported within the OCI, but at this point, the only
+active and viable reference implementation within the OCI is the `runc`
+implementation of the runtime specification, based around the contributed
+**libcontainer** codebase contributed by Docker.
+ - [runc](https://github.com/opencontainers/runc)
+
+Runc is also unique in that it is an open source reference implementation,
+but also a core dependent ecosystem component underneath the majority of
+container engine implementations in use today.
+
+For any future reference implementation to be adopted by the OCI, it would
+need to be kept in sync with the specification it implements. For a change
+to be accepted to the spec, the equivalent implementation would need to be
```

Comment:
I think it'd be worth articulating the converse here as well; for a change to be accepted to the reference implementation, the equivalent change would need to be adopted in the spec as well.  Runc hasn't followed this, and that means that integrators with runc may be integrating with its specific behavior rather than with the behavior defined by the spec.

Reply:
Maybe, but there is a necessary balance here.

It would be a mistake to accept changes into a specification if they haven't been tested "in the wild" (or at the very least, implemented at least once). The image-spec had this exact problem -- no reference implementation (at least, until umoci) made it so that we had to do some pretty large reworks during the release-candidate phase of `1.0`.

That being said, I think that we made too much of a concession to runc in the runtime-spec (though I would argue this is also historically driven -- runc was ready and in wide use quite a while before the spec was). There are many things in runc which really should be in the spec, and I am constantly frustrated to see that many implementations of the runtime-spec spend more of their time reading the runc source code than they do the spec.

Maybe we should make it so that no *release* of the project should have any major not-in-the-spec features, which would allow for experimentation and development without risking people becoming dependent on the feature. Unfortunately runc's behaviour doesn't really break this rule (because it breaks the entire concept of releases given that we've been doing `1.0` release candidates for 4 years now) -- but I think it's fair to say that this is something which no other OCI project will or should repeat.

Reply:
Given this section was updated due to further review below from @cyphar, there is no more commentary on spec vs. ref. implementation alignment. Not that it isn't an important topic in general, but maybe it didn't fit in this document anyway? Feedback welcome.